vagrant/roles/local.prep/tasks/main.yml

```diff
@@ -38,5 +38,5 @@
 - name: modify ssh-config-setup
   replace:
     path: ./ansible/ssh-config-setup
-    regexp: "IdentityFile .*\\.vagrant\\.d/insecure_private_key"
+    regexp: "IdentityFile .*\\.vagrant\\.d/insecure_private_key\""
```

Comment:
This is surprising.
I don't remember there being a trailing space in my case.
I will test it out locally.